ts/features/itwallet/common/components/ItwCredentialCard/ItwCredentialCard.tsx

```diff
@@ -69,11 +95,11 @@ export const ItwCredentialCard = ({
       };
     }
 
-    return tagPropsByStatus[credentialStatus];
-  }, [credentialStatus, needsItwUpgrade]);
+    return tagPropsByStatus[status];
+  }, [status, needsItwUpgrade]);
 
   const { titleColor, titleOpacity, colorScheme } = useMemo<StyleProps>(() => {
-    const isValid = validCredentialStatuses.includes(credentialStatus);
+    const isValid = validCredentialStatuses.includes(status);
```

Comment:
Instead of extending the if case, we could add the status which is considered valid here. What do you think?
```suggestion
    // Explain why we added jwtExpired as valid:
    const isValid = [...validCredentialStatuses, "jwtExpired"].includes(status);
```

Reply:
Thanks for the suggestion, addressed in [c77b53d](https://github.com/pagopa/io-app/pull/7491/commits/c77b53d833aadb16e168ed84df25a03bcad63111)